mmv1/products/datastream/ConnectionProfile.yaml

```
@@ -466,6 +478,136 @@ properties:
         description: |
           Database for the SQL Server connection.
         required: true
+  - name: 'mongodb_profile'
```

Names here should use camelCase.

```suggestion
  - name: 'mongodbProfile'
```